src/Schema/Directives/AggregateDirective.php

```diff
@@ -96,6 +96,21 @@ enum AggregateFunction {
 
     public function resolveField(FieldValue $fieldValue): FieldValue
     {
+        $modelArg = $this->directiveArgValue('model');
+        if (is_string($modelArg)) {
+            $fieldValue->setResolver(function ($root, array $args, GraphQLContext $context, ResolveInfo $resolveInfo) use ($modelArg) {
+                /** @var EloquentBuilder $query */
```

Comment:
use `assert`

Reply:
https://www.php.net/manual/de/function.assert.php over `@var`

Reply:
Thanks done and pushed